_faq/gsoc-requirements.md

```diff
@@ -4,6 +4,7 @@ question: "What are the requirements to be accepted for GSoC?"
 Other than the [requirements from Google](https://developers.google.com/open-source/gsoc/faq#what_are_the_eligibility_requirements_for_participation) you will have to meet the following:
 
 * You have completed the [coala Newcomers guide](https://coala.io/newcomer)
-* You have a submission to [our usability survey](https://coala.io/usability)
-	so we know that you know how to *use coala*.
-	
\ No newline at end of file
+* You have a submission to [our usability survey](https://coala.io/usability) 
+    you know how to *use coala*.
```

Comment:
you have changed the words here.  and the first word on this line should be under "You" on the previous line.